src/features/linter-provider.ts

```diff
@@ -37,6 +37,12 @@ export default class FortranLintingProvider {
       ...process.env,
       LC_ALL: 'C'
     };
+    if (process.platform == 'win32') {
+      // Windows needs to know the path of other tools
+      if (!env.Path.includes(path.dirname(command))) {
+        env.Path = `${path.dirname(command)}${path.delimiter}` + env.Path;
```

Comment:
You can make it like this 

```suggestion
        env.Path = `${path.dirname(command)}${path.delimiter}${env.Path}`
```